### PR TITLE
Enabled Managed prometheus metrics

### PR DESCRIPTION
**Pull request by datsabk**
sample config: 
```
- type: amp
    regions:
      - eu-west-1
    metrics:
      - name: DiscardedSamples
        statistics:
        - Sum
        period: 60
        length: 600
```
Amazon Managed Prometheus is a service gradually picking pace. At my current organization, we use over 50 AMP workspaces and getting metrics for them is crucial. 

I am also looking to get `AWS/Usage` metrics which should be covered in [this](https://github.com/nerdswords/yet-another-cloudwatch-exporter/pull/623) PR. Collectively, they will meet the requirement for [this](https://github.com/nerdswords/yet-another-cloudwatch-exporter/issues/674) enhancement

### CODE REVIEW

**Comment by datsabk** [comment]
@cristiangreco - Requesting review for the PR.

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/nerdswords/yet-another-cloudwatch-exporter
 go 1.18
 
 require (
-	github.com/aws/aws-sdk-go v1.44.96
+	github.com/aws/aws-sdk-go v1.44.108
```
Comment (cristiangreco) [inline go.mod:6]: Is this needed for any particular feature? Otherwise I'd leave dependecy upgrade out of the PR.

Comment (datsabk) [reply go.mod:6]: It's a minor version upgrade but not a mandatory requirement for it

**Comment by datsabk** [comment]
@cristiangreco - I see the upgrade of SDK already went through via dependabot - You can proceed with the further review of the PR. I've pulled the latest master branch into this.

**Comment by datsabk** [comment]
@cristiangreco @thomaspeitz - The SDK upgrade is no longer part of the PR. Could you please review and help in getting this merged.

**Comment by datsabk** [comment]
@cristiangreco thanks for looking into this. Can we get a OWNERS.md added for future to get the attention of right folks here. 

Thanks

**Comment by cristiangreco** [comment]
Hi @datsabk, sorry I'm not sure what would be OWNERS.md used for?

**Comment by datsabk** [comment]
@cristiangreco OWNERS.md tells who owns the repository. It allows folks to easily address owner when needed. 

Additionally a MAINTAINERS.md file could have list of people who can actively maintain this tool to take the development ahead at a faster pace.

File: pkg/sessions.go [hunk 2]
```diff
@@ -457,6 +478,22 @@ func createEC2Session(sess *session.Session, region *string, role Role, fips boo
 	return ec2.New(sess, setSTSCreds(sess, config, role))
 }
 
+func createPrometheusSession(sess *session.Session, region *string, role Role, fips bool, isDebugEnabled bool) prometheusserviceiface.PrometheusServiceAPI {
+	fmt.Println("Creating prometheus sessions")
```
Comment (z0rc) [inline pkg/sessions.go:482]: This is spamming yace logs and breaks structured json log parsing.

Comment (cristiangreco) [reply pkg/sessions.go:482]: Fixed in https://github.com/nerdswords/yet-another-cloudwatch-exporter/pull/715

**Pull request merged**